rpcs3/Emu/Audio/Cubeb/CubebBackend.h

```diff
@@ -57,6 +57,13 @@ class CubebBackend final : public AudioBackend
 
 	void CloseUnlocked();
 
-	std::tuple<cubeb_devid, std::string /* dev_ident */, u32 /* ch_cnt */> GetDevice(std::string_view dev_id = "");
-	std::tuple<cubeb_devid, std::string /* dev_ident */, u32 /* ch_cnt */> GetDefaultDeviceAlt(AudioFreq freq, AudioSampleSize sample_size, AudioChannelCnt ch_cnt);
+	struct device_handle
+	{
+		cubeb_devid handle{};
```

Comment:
Is this a typo and should say cubeb_device handle{};
Or would maybe cubeb_device.id  handle{}; be better?
I noticed in cpp you changed some dev_id to device.id for improved readability.

Reply:
cubeb_devid is a cubeb type which is actually just an obscure pointer used as handle.
So no. It's not a typo